library/src/main/java/com/novoda/downloadmanager/lib/FileDownloadInfo.java

```diff
@@ -356,6 +356,10 @@ private void clearHeaders() {
         requestHeaders.clear();
     }
 
+    public boolean isResumable() {
+        return alwaysResume || (eTag != null && isNoIntegrity());
```

Comment:
is `isNoIntegrity` used now outside the class? thinking of reducing visibility of the method